.github/workflows/chart-release.yml

```diff
@@ -5,6 +5,7 @@ on:
       - 'charts/**'
     branches:
       - develop
+      - "release/**"
```

Comment:
First time we push to a release/xxx, the version is still a -develop, do we need to block it here?

Reply:
I have included "blocking publish" as safety measure.